docs/bonding/index.adoc

```diff
@@ -105,11 +106,11 @@ If the value of ETH crosses a security threshold, open _Deposit_ s will enter
 
 === BTC price drop relative to ETH
 
-Since <<Signer Fees>> are denominated per BTC in custody (with
-overcollateralization factored in), a BTC value drop against the
-bonded asset translates in lower fees for Signers. Note that this does not
-create any issue for TBTC reserves, but it makes the system less attractive to
-signers looking to earn interest via fees on their assets.
+Since <<signer-fees/index.adoc#signer-fees,signer fees>> are denominated per
+BTC in custody (with overcollateralization factored in), a BTC value drop
+against the bonded asset translates in lower fees for Signers. Note that this
+does not create any issue for TBTC reserves, but it makes the system less
+attractive to signers looking to earn interest via fees on their assets.
```

Comment:
You didn't do this, but we shouldn't call it "interest" as it isn't... maybe just "attractive to signers earning fees on their assets"? Anyway fine to kick, nit

Reply:
Fixed.